packages/mobile/src/identity/verification.ts

```diff
@@ -160,6 +160,11 @@ export function* doVerificationFlow() {
 
       yield put(setVerificationStatus(VerificationStatus.RequestingAttestations))
 
+      // Account needs to be unlocked to submit attestations
+      const unlocked: boolean = yield call(unlockAccount, account)
+      if (!unlocked) {
+        throw new Error(ErrorMessages.INCORRECT_PIN)
+      }
```

Comment:
This addition isn't necessary because we call `getConnectedUnlockedAccount` above so the account is already unlocked.

Reply:
Should we change the above to `getConnectedAccount`, and only unlock if `!status.isVerified`? It seems that if the user is using DEK and the account is already verified, then it's unnecessary to request account unlock

Reply:
I did hit it but only when I intentionally waited 5 minutes.. I'll remove the changes - less risk and we can re-add if it's an issue.